### ISSUE TITLE
NRE when using FTS query on an object that also has a spatial field

### ISSUE THREAD

**Issue by DSilence** [aspnet/EntityFrameworkCore#13686]
When doing a FTS query against an entity which has a spatial field, NRE is observed (in SqlServerGeometryMethodTranslator).

```
Stack trace:
System.NullReferenceException: Object reference not set to an instance of an object.
  at at Microsoft.EntityFrameworkCore.SqlServer.Query.ExpressionTranslators.Internal.SqlServerGeometryMethodTranslator.Translate(MethodCallExpression methodCallExpression)
  at at Microsoft.EntityFrameworkCore.Query.ExpressionTranslators.RelationalCompositeMethodCallTranslator.<>c__DisplayClass6_0.<Translate>b__0(IMethodCallTranslator translator)
  at at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
  at at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Func`2 predicate, Boolean& found)
  at at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
  at at Microsoft.EntityFrameworkCore.Query.ExpressionTranslators.RelationalCompositeMethodCallTranslator.Translate(MethodCallExpression methodCallExpression, IModel model)
  at at Microsoft.EntityFrameworkCore.Query.ExpressionVisitors.SqlTranslatingExpressionVisitor.VisitMethodCall(MethodCallExpression methodCallExpression)
  at at System.Linq.Expressions.MethodCallExpression.Accept(ExpressionVisitor visitor)
  at at System.Linq.Expressions.ExpressionVisitor.Visit(Expression node)
  at at Remotion.Linq.Parsing.ThrowingExpressionVisitor.Visit(Expression expression)
  at at Microsoft.EntityFrameworkCore.Query.ExpressionVisitors.SqlTranslatingExpressionVisitor.Visit(Expression expression)
  at at Microsoft.EntityFrameworkCore.Query.RelationalQueryModelVisitor.VisitWhereClause(WhereClause whereClause, QueryModel queryModel, Int32 index)
  at at Remotion.Linq.Clauses.WhereClause.Accept(IQueryModelVisitor visitor, QueryModel queryModel, Int32 index)
  at at Remotion.Linq.QueryModelVisitorBase.VisitBodyClauses(ObservableCollection`1 bodyClauses, QueryModel queryModel)
  at at Remotion.Linq.QueryModelVisitorBase.VisitQueryModel(QueryModel queryModel)
  at at Microsoft.EntityFrameworkCore.Query.EntityQueryModelVisitor.VisitQueryModel(QueryModel queryModel)
  at at Microsoft.EntityFrameworkCore.Query.RelationalQueryModelVisitor.VisitQueryModel(QueryModel queryModel)
  at at Microsoft.EntityFrameworkCore.Query.EntityQueryModelVisitor.CreateQueryExecutor[TResult](QueryModel queryModel)
  at at Microsoft.EntityFrameworkCore.Storage.Database.CompileQuery[TResult](QueryModel queryModel)
  at at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.CompileQueryCore[TResult](Expression query, IQueryModelGenerator queryModelGenerator, IDatabase database, IDiagnosticsLogger`1 logger, Type contextType)
  at at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.<>c__DisplayClass13_0`1.<Execute>b__0()
  at at Microsoft.EntityFrameworkCore.Query.Internal.CompiledQueryCache.GetOrAddQueryCore[TFunc](Object cacheKey, Func`1 compiler)
  at at Microsoft.EntityFrameworkCore.Query.Internal.CompiledQueryCache.GetOrAddQuery[TResult](Object cacheKey, Func`1 compiler)
  at at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.Execute[TResult](Expression query)
  at at Microsoft.EntityFrameworkCore.Query.Internal.EntityQueryProvider.Execute[TResult](Expression expression)
  at at Remotion.Linq.QueryableBase`1.GetEnumerator()
  at at System.Collections.Generic.List`1.AddEnumerable(IEnumerable`1 enumerable)
  at at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
  at at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
  at Repro.Program.Main(String[] args) in C:\Projects\Temp\Repro\Repro\Program.cs:65

```

### Steps to reproduce
Minimal repro project here [Repro.zip].(https://github.com/aspnet/EntityFrameworkCore/files/2494989/Repro.zip). 

### Further technical details
EF Core version: 2.2.0-preview3-35497
Database Provider: Microsoft.EntityFrameworkCore.SqlServer
Operating system: Windows 10 1809
IDE: Rider 2018.3 EAP3

**Comment by ajcvickers** [comment]
@bricelam to take a look

**Comment by bricelam** [comment]
Fixed by PR #13648. Try [the nightly builds](https://dotnet.myget.org/gallery/dotnet-core).
